app/views/requests/_letter_of_consent.html.erb

```diff
@@ -1,5 +1,14 @@
-<%= f.govuk_file_field :letter_of_consent, label: nil, hint: { text: simple_format(t("helpers.hint.request_form.letter_of_consent")) } %>
-<p class="govuk-body-m"><%= t("helpers.hint.request_form.letter_of_consent_file_size") %></p>
+<span class="govuk-body">You must provide a letter of consent from the subject of this request which:</span>
+<ul class="govuk-list govuk-list--bullet" style="margin-bottom:0;">
+  <li>States that they consent to their information being released to you</li>
+  <li>Is signed by them</li>
+  <li>Is dated within the last 6 months</li>
+</ul>
+<p class="govuk-body-m">There is no official template.</p>
 
+<p class="govuk-body-m"><strong>Letter of consent</strong></p>
+<p class="govuk-hint"><%= t("helpers.hint.request_form.letter_of_consent_file_size") %></p>
```

Comment:
These should be defined as the label and hint for the letter_of_consent. Make sure you set the label to small as in other forms to get the bold version.

Reply:
I have changed to below but the font has changed.
<p class="govuk-label--s">Letter of consent</p>
<p class="govuk-hint"><%= t("helpers.hint.request_form.letter_of_consent_file_size") %></p>